Docs for october 4960 (#439)
octobercms/docs: themes-development.md

        - Acme.Blog

When the theme is installed for the first time, the system will attempt to install the required plugins at the same time.

<a name="localization"></a>
## Localization

Themes can provide backend localization keys through files placed in the **lang** subdirectory of the theme's directory. These localization keys are registered automatically only when interacting with the October backend and can be used for form labels just like [plugin localization](../plugin/localization)
   
> **Note**: Translating frontend content should be handled with the [RainLab.Translate](https://octobercms.com/plugin/rainlab-translate) plugin.

> <a name="file-structure"></a>

Reviewer: ```suggestion
<a name="localization-file-structure"></a>
```

        - Acme.Blog

When the theme is installed for the first time, the system will attempt to install the required plugins at the same time.

<a name="localization"></a>
## Localization

Themes can provide backend localization keys through files placed in the **lang** subdirectory of the theme's directory. These localization keys are registered automatically only when interacting with the October backend and can be used for form labels just like [plugin localization](../plugin/localization)
   

> > **Note**: For translating front-end content, [there are plugins that can be used](http://octobercms.com/plugin/rainlab-translate) for this purpose.

Reviewer: ```suggestion
> **Note**: Translating frontend content should be handled with the [RainLab.Translate](https://octobercms.com/plugin/rainlab-translate) plugin.
```

<a name="file-structure"></a>
### Localization directory and file structure

Below is an example of the theme's lang directory:

    themes/
      acme/               <=== Theme directory
        lang/             <=== Localization directory
          en/             <=== Language directory
            lang.php      <=== Localization file
          fr/
            lang.php

The **lang.php** file should define and return an array of any depth, for example:

    <?php return [
        'options' => [
            'website_name' => 'OctoberCMS'
        ]
    ];

> > **Note**: the full localization key format is the following: theme.theme-code::lang.key. So for example above it
> > would be `theme.theme-code::lang.options.website_name`

Reviewer: ```suggestion
You are then able to reference the keys using `themes.theme-code::lang.key`. In the above example, the full language key you would use to reference the "website_name" localization key would be `themes.acme::lang.options.website_name`
```

        - Acme.Blog

When the theme is installed for the first time, the system will attempt to install the required plugins at the same time.

<a name="localization"></a>
## Localization

> Themes can have localization files in the **lang** subdirectory of the theme directory. Theme's localization files
>  are registered automatically when the theme is being edited in the backend. The localization strings are supported
>   automatically in the back-end customization form labels - if you provide the localization key instead of a real
>    string, the system will try to load it from the localization file.

Reviewer: ```suggestion
Themes can provide backend localization keys through files placed in the **lang** subdirectory of the theme's directory. These localization keys are registered automatically only when interacting with the October backend and can be used for form labels just like [plugin localization](../plugin/localization)
```

When the theme is installed for the first time, the system will attempt to install the required plugins at the same time.

<a name="localization"></a>
## Localization

Themes can have localization files in the **lang** subdirectory of the theme directory. Theme's localization files
 are registered automatically when the theme is being edited in the backend. The localization strings are supported
  automatically in the back-end customization form labels - if you provide the localization key instead of a real
   string, the system will try to load it from the localization file.
   
> **Note**: For translating front-end content, [there are plugins that can be used](http://octobercms.com/plugin/rainlab-translate) for this purpose.

<a name="file-structure"></a>
### Localization directory and file structure

Below is an example of the theme's lang directory:

    themes/
      acme/               <=== Theme directory
        lang/             <=== Localization directory
          en/             <=== Language directory
            lang.php      <=== Localization file
          fr/
            lang.php

The **lang.php** file should define and return an array of any depth, for example:

>     <?php
> 
>     return [

Reviewer: ```suggestion
    <?php return [
```
